pom.xml

```diff
@@ -152,20 +162,20 @@
                         <!-- config is inherently not testable -->
                         <exclude>org/sagebionetworks/bridge/udd/config/*</exclude>
 
+                        <!-- no point in testing exceptions, since they are entirely boilerplate code -->
+                        <exclude>org/sagebionetworks/bridge/udd/exceptions/*</exclude>
```

Comment:
Yes, but testing exception constructors and that they properly pass through to the super class and then enforcing coverage metrics on them isn't a productive use of our time.